images/macos/tests/Powershell.Tests.ps1

```diff
@@ -38,4 +40,10 @@ Describe "Powershell" {
             }
         }
     }
+    
+    Context "Azure CLI" {

```

Comment:
Why is az cli related to PowerShell section?

Reply:
Az CLI installs in powershell.sh script. I think it's the best option to run it together with powershell itself.